docs/raspberry.md

```diff
@@ -0,0 +1,74 @@
+# Installing on a Raspberry Pi
+
+!!! Warning
+	This process currently only installs the requirements to run the bot live. You will need to do the backtesting and optimising on different (more powerful) hardware.
+
+!!! Info	
+	Tested on a Raspberry Pi 3b with the latest Raspberry Pi OS.
```

Comment:
Please specify the version of raspbian it is tested on. 'The latest' will have a different meaning in 6 months from now :)